secondarylib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala

```diff
@@ -80,15 +80,12 @@ class SecondaryWatcher[CT, CV](universe: => Managed[SecondaryWatcher.UniverseTyp
           // want to trigger event message only when there is no error.
           unclaimDataset(u, secondary, job, sendMessage = true, startingMillis)
         } catch {
-          case ex: Exception =>
+          case ex: Throwable =>
             try {
```

Comment:
I _think_ you could still catch an Exception here and just ignore the Throwable not an Exception case. but okay